_posts/2018-10-30-conducting-good-retrospectives.md

```diff
@@ -0,0 +1,110 @@
+
+The primary motivation for the technique to is **avoid interruptions** and **talking over each other**.
+
+#### Format/Activity
+
+Free form conversation is not conducive to reaching conclusions, so it helps to pick a meeting format that fits well with your team. At the beginning of the meeting, the facilitator will explain the format and confirm that everyone is okay with that decision. Different formats optimize for different aspects of your team: project performance, interpersonal communication, or general frustration.
+
+Picking a format gives the **meeting structure** and often an **objective**, allowing the team to work together towards a known and common goal.
+
+#### Action Items
+
+A necessary output of the meeting is a list of action items. An action item is a task that requires further action in order to accomplish it and is assigned to a specific person. You've probably experienced the situation where everyone agrees something should get done, but no one ever does it.
+
+Sometimes you run into an action item that isn't a specific task, but possibly a general change in behavior that is meant to be actioned by the whole team. In this case, the real action item would be bearing the responsibility of shepherding the adoption of this new behavior for a couple weeks.
+
+Leaving the meeting with a list of action items ensures that the tasks you spent an hour finding, **actually get done**.
+
+## Case Study
+
+For the last year, my team has had a weekly retro, alternating the format each week. We operate in two week sprints, so in the middle of the sprint we typically concentrate on interpersonal issues or other general frustrations and at the end of the sprint we focus on project-specific issues.
+
+However, at the beginning of any retrospective, we solicit the opportunity to suggest an alternative format.
+
+#### Mid-Sprint Retrospective
+
+During the Mid-Sprint Retrospective, we engage in the [Lean Coffee](http://leancoffee.org) activity (unless someone suggests an alternate activity!). The chosen facilitator will explain the chosen activity and pass out the required supplies. 
+
+The basics of the Lean Coffee activity are:
+
+1. Generate discussion topics
+    - Limit to 5 minutes
+1. Vote on topics
+1. Discuss topics
+    - Limit to 5 minutes, then vote on continuing with a 2 minute extension, or else move on to next topic.
+    - Facilitator will take notes and track action items.
+
+Any topics that aren't discussed are carried over to the next meeting, unless the author states otherwise.
+
+#### Sprint Retrospective
+
+During the Sprint Retrospective we use the [Four Quadrant](http://www.funretrospectives.com/lessons-learned-quadrants-planning-vs-success/) activity. As with the Mid-Sprint retro, the chosen facilitator will explain the activity and pass out the required supplies.
```

Comment:
The grid that we do isn't exactly the Four Quadrant activity. The axes are different and the execution is different. We can chat in-person if you'd like.

When I first ran the grid thing, part of the activity was supposed to be the group collaborating on where to put the various event-stickies on a given axis. I think that part has been lost over the last couple of years. I wonder if it was worthwhile...

Incidentally this paragraph gave me a neat idea for a different retro activity. Remind me to talk to you later. :smile:

Reply:
Interesting, I did notice that the activity described in that link wasn't exactly like how I described it, but I figured it was similar enough to keep the link. 

I think I'll add a bit about how this is a slightly different take on that activity

Reply:
> the group collaborating on where to put the various event-stickies on a given axis

It's a pretty time consuming part of the activity, and most people seemed to feel it didn't add much value.

Reply:
https://medium.com/serious-scrum/adding-another-dimension-to-the-traditional-scrum-retrospective-format-6014d89ebebb